static/src/javascripts/projects/common/modules/commercial/creatives/fabric-v1.js

```diff
@@ -80,13 +80,13 @@ define([
                 bonzo(this.layer2).css('background-position', '');
             }
 
-            if (this.scrollType === 'fixed' && hasScrollEnabled) {
+            if (this.scrollType === 'fixed') {
```

Comment:
Might want to add a `hasBackgroundFixedsupport` check to this, too - otherwise, if an Android phone starts supporting the property, every scroll will shift the background twice, and your fixed backgrounds will suddenly become parallax backgrounds!